sdk/translation/azure-ai-translation-document/tests/test_translation.py

```diff
@@ -32,7 +32,10 @@ class TestTranslation(DocumentTranslationTest):
     def test_active_directory_auth(self):
         token = self.generate_oauth_token()
         endpoint = self.get_oauth_endpoint()
-        client = DocumentTranslationClient(endpoint, token)
+        kwargs = {}
+        if os.getenv("AZURE_AUTHORITY_HOST", "").find("ppe") != -1:
+            kwargs["credential_scopes"] = ["https://ppe.cognitiveservices.azure.com/.default"]
```

Comment:
An alternative would be to set the credential scope as an env var in the subscription config or as an output in the arm template, that way you don't have to hardcode endpoints into the test.

Reply:
Great idea, going to update to do that.